Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/math/Math.kt

```diff
@@ -0,0 +1,9 @@
+package de.rki.coronawarnapp.util.math
+
+import kotlin.math.roundToInt
+
+/**
+ * Round a fraction to a decimal number with the required decimal places
+ * @param denominator Power of 10
+ */
+fun Double.roundToDecimal(denominator: Double = 100.0): Double = (this * denominator).roundToInt() / denominator
```

Comment:
I think `denominator` should not have a default, because any default always depends on the context.

Second, the current default is technically not correct. It correctly rounds 3.599 to 3.6 (actually 3.60, but 0 is suppressed) but it would round 3.559 to 3.56.

Instead, I would suggest to accept an (unsigned) int as number decimal places and calculate the denominator as 10**decimalPlaces.

Reply:
Adjusted as per suggestion